tests/integration/test_session.py

```diff
@@ -129,6 +129,45 @@ def scorer(name, proposal):
         assert best['name'] == 'a_tunable'
         assert best['config'] == {'a_parameter': 2}
 
+    def test_normalized_score_becomes_none(self):
```

Comment:
@pvk-developer Can you add a doscstring here explaining the rationale of this test case? Others are quite self-explaining, but this one can be tricky to understand by only reading at the code.